testdata/config/azure_key_huge.yaml

```diff
@@ -5,6 +5,7 @@ logging:
 
 components:
   - libfuse
+  - stream
```

Comment:
Should we also keep a file cache only test as well?

Reply:
Yes its a mix, some of the config files in testdata have stream while some dont. for e.g. key based regular test is file-cache only while sas is with stream+file-cache. Huge file tests are with stream+